src/Models/ExternalLogs.php

```diff
@@ -160,12 +164,17 @@ private function addLogWithMetrics(string $logType, string $message, string $typ
    * @param string $level
    * @param string $message
    */
-  private function addCustomLog(string $level, string $message) {
-    $this->logs[] = [
+  private function addCustomLog(string $level, string $message, $details = null) {
+    $log = [
```

Comment:
Am I missing something or shouldn't this get added to $logs?

Reply:
Good catch. I guess I never finished my thought here.